modules/gapi/perf/common/gapi_imgproc_perf_tests_inl.hpp

```diff
@@ -476,15 +476,15 @@ PERF_TEST_P_(SobelPerfTest, TestPerformance)
 
     // G-API code //////////////////////////////////////////////////////////////
     cv::GMat in;
-    auto out = cv::gapi::Sobel(in, dtype, dx, dy, kernSize );
+    auto out = cv::gapi::Sobel(in, dtype, dx, dy, kernSize);
     cv::GComputation c(in, out);
 
     // Warm-up graph engine:
     c.apply(in_mat1, out_mat_gapi, std::move(compile_args));
 
     TEST_CYCLE()
     {
-        c.apply(in_mat1, out_mat_gapi, std::move(compile_args));
+        c.apply(in_mat1, out_mat_gapi);
```

Comment:
@dmatveev Is it correct change?

Reply:
Yes and no. since `compile_args` are moved twice, the latter call to apply in fact passes an empty array - which has no effect (the underlying graph is not recompiled).

Reply:
It works now but our implementation of `apply()` doesn't check contents of compile args so if we fix this problem, code like this will break. The issue is known and tracked internally alreday